redaxo/src/addons/cronjob/lib/manager.php

```diff
@@ -111,7 +111,7 @@ public function log($success, $message)
                 $name = '[no name]';
             }
         }
-        $log = new rex_log_file(rex_path::addonData('cronjob', 'cronjob.log'), 2000000);
+        $log = new rex_log_file(rex_path::log('cronjob.log'), 2000000);
```

Comment:
Somit mûssen wir die min rex version anheben, oder?

Reply:
Ja, mache ich im Release-PR.

(Kann es hier nicht machen, da man nach merge dann cronjob nicht mehr installieren könnte, da die core-version noch auf 5.8.1 steht. Zu der Thematik werden ich mich nach dem Release aber noch mal melden, möchte da was ändern in unserem Prozess.)